src/accessibility/KeyboardShortcuts.ts

```diff
@@ -392,6 +395,13 @@ export const KEYBOARD_SHORTCUTS: IKeyboardShortcuts = {
         },
         displayName: _td("Navigate to previous message in composer history"),
     },
+    [KeyBindingAction.ShowStickerPicker]: {
+        default: {
+            ctrlOrCmdKey: true,
+            key: Key.COMMA,
+        },
+        displayName: _td("Send a sticker"),
+    },
```

Comment:
This shortcut is already taken

https://github.com/matrix-org/matrix-react-sdk/blob/a58b1e9d792c43c9dc1330ce448cb44e7401e95a/src/accessibility/KeyboardShortcuts.ts#L632-L638

Reply:
(metaKey = CMD on macOS)

Reply:
Aieeeee